### ISSUE TITLE
blacke3's method has 'multithreading' param missing since blake3 v0.3 

### ISSUE THREAD

**Issue by davidandreoletti** [oconnor663/blake3-py#29]
Bug description

```
# v0.2.1: Creates a blake3 object
blake3(value.encode("utf-8"), multithreading=False)
```

```
# v0.3.0: Fails to create a blake3 object
blake3(value.encode("utf-8"), multithreading=False)
# Exception raised: blake3.__new__() got an unexpected keyword argument 'multithreading'

```

How to reproduce:

- Install blake3 v0.3
- Run sample code

**Comment by davidandreoletti** [comment]
Never mind, `multithreading` param has changed to `max_threads` since https://github.com/oconnor663/blake3-py/commit/a9cf2204e020b55ff4c27f304fa4357bc62cab56#diff-fd6bdf11107bc98183eaa8a18856e20d5b13ad13fe7d80a916a99fedfbccbf80

**Comment by oconnor663** [comment]
Yep. For more details you can read the release notes here: https://github.com/oconnor663/blake3-py/releases

This project uses semver (or at least the conventional extension of semver to pre-1.0 versions), so all the releases in the 0.3.x series will be backwards-compatible with 0.3.0, and the next compatibility break will be released as 0.4.0.
